README.md

```diff
@@ -129,6 +129,41 @@ Below is a rough sketch of potential interfaces.
   The environment record does not contain a property for any names that are
   imported and reexported without a lexical binding.
 
+A loader privately retains references to:
+
+* A global object
+* A global environment record
+* A static module record promise memo
+* A static module record memo
+* A module instance promise memo
```

Comment:
what is it? TLA?

Reply:
Not sure what you mean by TLA (three letter acronym and translation look-aside are my usual expansions!) But, `Map<FullSpecifierString, Promise<ModuleInstance>>` is what I intend here. We need both synchronous and asynchronous maps because we need to support both synchronous initialization of partial subgraphs and top-level-await.

Reply:
Yes, I mean top-level await here.